src/redux/teachers/teachers.slice.ts

```diff
@@ -43,6 +43,18 @@ const teachersSlice = createSlice({
         state.isLoading = false
         state.error = null
       })
+      .addCase(bookLesson.pending, (state) => {
+        state.isLoading = true
+        state.error = null
+      })
+      .addCase(bookLesson.fulfilled, (state) => {
+        state.isLoading = false
+        state.error = null
+      })
+      .addCase(bookLesson.rejected, (state) => {
+        state.isLoading = false
+        state.error = null
+      })
```

Comment:
obviously useless part of code, because its not asynchronyus func, just do it in reducers